public/plugin.tsx

```diff
@@ -330,13 +344,23 @@ export class ObservabilityPlugin
       mount: appMountWithStartPage('notebooks'),
     });
 
-    core.application.register({
-      id: observabilityPanelsID,
-      title: observabilityPanelsTitle,
-      category: OBSERVABILITY_APP_CATEGORIES.observability,
-      order: observabilityPanelsPluginOrder,
-      mount: appMountWithStartPage('dashboards'),
-    });
+    if (core.chrome.navGroup.getNavGroupEnabled()) {
```

Comment:
Just for my knowledge, is this like a feature flag?

Reply:
Yes, the navigation change can be toggled, and the UI plan currently list regular dashboards as going into the same category. Until it is depreciated the idea was to rename ours for clarity while using the new navigation.